src/slugify.js

```diff
@@ -10,11 +10,11 @@ import * as _ from 'lodash';
  * @return {string} URL-safe form of a string.
  * @example
  *
- * _.slugify('A Test')
+ * _.slugify('A Test');
  * // => a-test
  */
 function slugify(string) {
-    return _.toString(string).trim().toLowerCase().replace(/ /g, '-').replace(/([^a-zA-Z0-9\._-]+)/, '');
+    return _.deburr(_.toString(string).trim().toLowerCase()).replace(/ /g, '-').replace(/([^a-zA-Z0-9\._-]+)/, '');
```

Comment:
With `deburr` added, maybe this should chained instead? Something like `_(string).toString().deburr().trim().toLower()`? `toString` maybe be unnecessary too, I would think lodash auto-converts to string for string operators.

Reply:
Ok, because lodash chains return on primitives, this would need to be `_(string).chain().deburr().trim().toLower().value()`.

Reply:
Thought about chaining but kind of complicated things with the .value() call and then the .trim.toLowerCase().replace()...

`_.chain(string).toString().deburr().value().trim().toLowerCase().replace(/ /g, '-').replace(/([^a-zA-Z0-9\._-]+)/, '');`

Without toString();

`_.deburr(string).trim().toLowerCase().replace(/ /g, '-').replace(/([^a-zA-Z0-9\._-]+)/, '');`

And yes, lodash converts it to a valid string.

Reply:
Oohhh right, forgot about lodash toLower().

Reply:
I think for now, since we're only using one lodash method, your solution is the shortest: `_.deburr(string).trim().toLowerCase()...`

If I ever need to add more lodash methods to the chain, I'll use an actual chain.

Reply:
So `_(string).chain().deburr().trim().toLower().value().replace(/ /g, '-').replace(/([^a-zA-Z0-9\._-]+)/, '');` will be.